package.json

```diff
@@ -51,7 +51,7 @@
     "axios": "~0.21.1",
     "bignumber.js": "~9.0.1",
     "chai-as-promised": "~7.1.1",
-    "ethers": "~5.5.1",
+    "ethers": "~5.4.5",
```

Comment:
actually a question

Reply:
I was thinking we updating package, but we downgrading it, so this should be hotfix. 5.5.1

Reply:
sorry :]

Reply:
No problem

Reply:
> I was thinking we updating package, but we downgrading it, so this should be hotfix. 5.5.1

I think it's not a hotfix, and it's actually a new feature.
Since the breaking changes are happening only in this branch, there's no reason to consider this a hotfix

Reply:
5.5.0 is not working, fix is downgrade ether package => its a hotfix

let's do this:
- hofix 5.5.1 only for downgrade ethers
- 5.6.0 feature release